R/tutorial.R

```diff
@@ -0,0 +1,44 @@
+#' Create a learnr tutorial.
+#'
+#' Creates a new tutorial in `inst/tutorials`. Tutorials are interactive R
+#' Markdown documents built with the `learnr` package.
+#'
+#' @section General setup:
```

Comment:
Can you think of a different heading here? This somehow made me expect a description of setup *I* needed to do versus the setup I can expect this function to do. It's probably also OK to let this be a bullet list in the first/main/only paragraph of description. In that case, you can solve the heading dilemma by deleting it 😁